neqo-bin/src/server/http09.rs

```diff
@@ -190,7 +190,6 @@ impl HttpServer {
                         .unwrap();
                     qdebug!("Wrote {}", sent);
                     *offset += sent;
-                    self.server.add_to_waiting(conn);
```

Comment:
No need to tell `Server` that a `Connection` might be waiting to be processed. `Server` will simply always `process` each `Connection`.